app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,77 @@
+class RecipesController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @recipes = Recipe.all
+  end
+

```

Comment:
- [ ] Currently, I can see other recipes as well which is not a public recipe. Please restrict and show only the recipe that has been created by the current user or is public recipe.